exercism/diamond/diamond.js

```diff
@@ -0,0 +1,44 @@
+var Diamond = function() {}
+
+Diamond.prototype.makeDiamond = function(letter) {
+  if (letter === 'A') {
+    return 'A\n'
+  }
+  var alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split('')
+  var character = ' '
```

Comment:
The variable 'character' is unnecessary.
This, I believe, is an instance where explicity passing in `' '` below during your join is better than passing in a variable.

Either than or don't name it "character", name it something like "whitespace"